integration-tests/mockApis/manageusers.js

```diff
@@ -1,7 +1,20 @@
 const { getFor, stubJson, getMatchingRequests, stubFor } = require('./wiremock')
 
 const stubUserMeRoles = (roles) => getFor({ urlPattern: '/users/me/roles', body: roles })
+
+const stubAExtSearchableRoles = ({
+  content = [
+    { roleCode: 'GLOBAL_SEARCH', roleName: 'Global Search' },
+    { roleCode: 'LICENCE_RO', roleName: 'Licence Responsible Officer' },
+    { roleCode: 'LICENCE_VARY', roleName: 'Licence Vary' },
+  ],
+}) =>
+  getFor({
+    urlPattern: '/externalusers/me/searchable-roles',
+    body: content,
+  })
 module.exports = {
```

Comment:
If you move this section directly into the module.exports (with the others) you don't need to export it.

Reply:
Moved